client/signup/steps/domains/style.scss

```diff
@@ -550,9 +588,119 @@ body.is-section-signup {
 		@include break-medium {
 			max-width: 290px;
 		}
+
+		@include break-large {
+			margin-left: 40px;
+		}
+
+		@include break-huge {
+			margin-left: 60px;
+		}
+
+		&.fade-out {
+			display: none;
+		}
 	}
 
-	.domains__domain-side-content.fade-out {
-		display: none;
+	.step-container-v2__heading {
+		h1:not(.small) {
+			letter-spacing: -0.4px;
+			/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+			line-height: 1.2em;
+		}
+
+		> p {
+			color: $gray-60;
+			font-size: 1rem;
+		}
+	}
+}
+
+.step-container-v2 {
+	&__content.step-container-v2__content--two-column-layout.domains__step-content.domains__step-content-domain-step {
+		column-gap: 0;
+		max-width: 1280px;
+	}
+
+	&__content-wrapper.padding {
+		padding: 0 12px;
+		gap: 0;
+	}
+
+	&__heading {
+		margin-top: 0;
+	}
+}
+
+.domains__step-content-domain-step > div {
+	&:nth-child(1) {
+		flex: 1 !important;
+	}
+
+	&:nth-child(2) {
+		display: flex;
+		flex-basis: auto !important;
+		flex-direction: column;
+		flex-grow: 0 !important;
+		flex-shrink: 1 !important;
+		flex-wrap: nowrap;
+	}
+}
+
+.domain-product-price {
+	&.domain-product-single-price.domain-product-price__domain-step-signup-flow {
+		align-self: center;
+	}
+
+	@include breakpoint-deprecated( "<660px" ) {
+		&.domain-product-price__domain-step-signup-flow {
+			flex-direction: column;
+		}
+	}
+}
+
+.domain-suggestion__content.domain-suggestion__content-domain {
+	align-items: unset;
+}
+
+body.is-section-stepper {
+	.featured-domain-suggestion.card .domain-product-price {
+		align-items: flex-start;
+		margin-top: 12px;
+	}
+
+	.formatted-header .formatted-header__title {
+		@media (min-width: 1080px) {
+			font-size: 2.75rem !important;
+		}
+	}
+
+	.register-domain-step__search.register-domain-step__search-domain-step:not(.is-sticky) {
+		@media (min-width: 661px) {
+			padding-top: 18px;
+		}
+	}
+
+	.onboarding:not(.is-onboarding-2023-pricing-grid) .step-container-v2__heading {
+		margin: calc(48px + 24px - var(--step-container-v2-top-bar-height)) 20px 24px 20px;
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: space-between;
+		width: unset;
+		align-items: center;
+		flex-direction: column;
+
+		@media (min-width: 960px) {
+			margin-top: calc(48px + 48px - var(--step-container-v2-top-bar-height));
```

Comment:
This is calculated based on the domain-only component - its margins/paddings are used to determine which should be the top here, along with the variable from the height of the different bar used in this flow.